test/project-config/forms/app/care_of_pregnant_women_and_others.xml

```diff
@@ -0,0 +1,2820 @@
+<?xml version="1.0"?>
```

Comment:
I wonder if we need `.xml` files here ? Since we will not be currently uploading to any instance ? What do you think ?

Reply:
No needed , I will juste remote it .